File: App_Santander_Test/app/src/main/java/com/ivan/bankapp/model/StatementList.java
package com.ivan.bankapp.model;

import com.google.gson.annotations.SerializedName;

public class StatementList {

    @SerializedName("statementList")
    private Statements[] statementList;

    public StatementList(Statements[] statementList) {
        this.statementList = statementList;
    }

    public Statements[] getStatementList() {
        return statementList;
    }

    public void setStatementList(Statements[] statementList) {
        this.statementList = statementList;
    }
}

File: app/src/main/java/br/com/testeandroidv2/model/statements/gson/StatementList.kt
package br.com.testeandroidv2.model.statements.gson

import android.os.Parcelable

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import kotlinx.android.parcel.Parcelize

@Parcelize
data class StatementList(@SerializedName("title") @Expose var title: String? = null,
                         @SerializedName("desc")  @Expose var desc: String? = null,
                         @SerializedName("date")  @Expose var date: String? = null,
                         @SerializedName("value") @Expose var value: Double? = null
) : Parcelable